source/administration/command-line-tools.md

```diff
@@ -27,9 +27,15 @@ _Available in Enterprise Edition E10 and higher_
 - Migrating sign-in options 
 - Resetting multi-factor authentication for a user 
 
+#### Using the CLI tool 
 
-Typing `platform -help` brings up the below documentation on usage. 
+Typing `platform -help` brings up documentation for the CLI tool.
 
+Notes: 
+- Parameters in CLI commands are order-specific.
+- Special characters (e.g. `!`) need to be escaped out (e.g. `/!`) if passed into parameters, like `-password`
```

Comment:
`|`, `(`, `)`, `\`, \`, and `"` all need to be escaped. Also, instead of having them escape individual characters, you could just say to surround the argument in single quotes like `mattermost -create_user -email asdf@example.com -password 'Secret!'`